tests/e2e/common-operator-integ-suite.sh

```diff
@@ -87,6 +108,8 @@ initialize_variables() {
   COMMAND="kubectl"
   ARTIFACTS="${ARTIFACTS:-$(mktemp -d)}"
   KUBECONFIG="${KUBECONFIG:-"${ARTIFACTS}/config"}"
+  LOCALBIN="${LOCALBIN:-"${HOME}/.local/bin"}"
+  OPERATOR_SDK="${LOCALBIN}/operator-sdk"
```

Comment:
Who is installing `operator-sdk` bin there?

Reply:
Is already present in the builder image that we use to run everything

Reply:
So, if the user want to execute the target without a container will need to have operator-sdk installed previously

Reply:
I'm not sure if I understand. Why creating `mkdir -p "${LOCALBIN}"` when `operator-sdk` should be already there? If it's not there, it does not make sense to create that dir anyway.

Reply:
Changed. It should be already there as you mention